src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +118,60 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
```

Comment:
Most of these already exist.

Also they should all use the existing extension slot.